.circleci/config.yml

```
@@ -194,7 +194,7 @@ jobs:
         environment:
           MYSQL_ROOT_PASSWORD:
           MYSQL_ALLOW_EMPTY_PASSWORD: yes
```

You can either choose to also change this line as follow:
```suggestion
          MYSQL_ALLOW_EMPTY_PASSWORD: true
```

... or postpone it for a future contribution.

---

Actually I removed it because it is already the default value.